utils/spdx/src/main/kotlin/model/SpdxFile.kt

```diff
@@ -99,10 +99,10 @@ data class SpdxFile(
     val licenseConcluded: String,
 
     /**
-     * The license information found in this file. To represent a not present value [SpdxConstants.NONE] or
-     * [SpdxConstants.NOASSERTION] must be used.
+     * The license information found in this file. According to the SPDX 2.3.1 spec, a not present value represents
```

Comment:
Note that [we officially only support SPDX 2.2](https://github.com/oss-review-toolkit/ort/blob/355f239c0200dfb11ff2b3f306257c8384ca505c/utils/spdx/src/main/kotlin/model/SpdxDocument.kt#L29), so quoting 2.3 specs here seems to be a bit misplaced.

Reply:
... and [the 2.2 specs](https://github.com/spdx/spdx-spec/blob/a05c12a2dd4652b1396fd2659f2cd3ea1f37faba/schemas/spdx-schema.json#L535) don't mention `NOASSERTION` explicitly.

That said, I'm personally with with making this default to `NOASSERTION`, but in the same run we should probably disallow the property to be empty, to always explicitly express the intent.

Reply:
SPDX 2.2.1 defines `NOASSERTION` for `licenseInfoInFiles` see https://github.com/spdx/spdx-spec/blob/development/v2.2.1/chapters/package-information.md#714-all-licenses-information-from-files-field-

SPDX 2.2.1 is the ISO version of the specification so I recommend we simply upgrade our SPDX generator from 2.2 to 2.2.1.

Reply:
> `licenseInfoInFiles` see https://github.com/spdx/spdx-spec/blob/development/v2.2.1/chapters/package-information.md#714-all-licenses-information-from-files-field-

@tsteenbe you're quoting "licenseInfo*In*Files" but linking to "licenseInfo*From*Files" docs, but these are two different fields, right?